Library/Homebrew/cmd/test-bot.rb

```diff
@@ -155,6 +156,14 @@ def run
           working_dir.cd { exec(*@command) }
         end
         write.close
+        if travis
+          print_thread = Thread.new do
+            while true do
+              printf "."
+              sleep 60
+            end
+          end
+        end
```

Comment:
Ok, please ignore my proposed implementation above. I was mistaking assuming `brew install` won't print anything until error. 

What I'm thinking I s I don't like print random dot if there is a output. And using thread to print dot could hang up Travis if building is truly freeze. 

How about we implementing this inside `Formula#system`? So it will only print dot when it should be.